pkg/psmdb/statefulset.go

```diff
@@ -419,7 +425,7 @@ func backupAgentContainer(cr *api.PerconaServerMongoDB, replsetName string, port
 			},
 			{
 				Name:  "PBM_MONGODB_URI",
-				Value: "mongodb://$(PBM_AGENT_MONGODB_USERNAME):$(PBM_AGENT_MONGODB_PASSWORD)@$(POD_NAME)",
+				Value: buildMongoDBURI(ctx, tlsEnabled, sslSecret),
```

Comment:
@gkech, we need to use the new value only in case of
`if cr.CompareVersion("1.20.0") >= 0 {`

please check the code how we use it

Reply:
I'll do, thanks Slava

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1835/commits/1a907dfccac5d96d212e7203517f0d82ed55ba82